app/src/main/res/layout/screen_teleconsult_prescription.xml

```diff
@@ -50,4 +51,28 @@
 
   </androidx.core.widget.NestedScrollView>
 
+  <LinearLayout
```

Comment:
I'm curious, why do we need this additional container?

If we need the additional frame with a different colour background, maybe `FrameLayout` would be a better choice.

Reply:
I'm also surprised that `ConstraintLayout` does not have a widget for this use case 😅

Reply:
Oh we can do that with `ConstraintLayout` it just requires usage of more widgets 😂 . So we can do something like setup barriers on top and bottom of the button and have a `View` with background color constrainted to those barriers.

Reply:
Right, should we use a `FrameLayout` then if all we want is just a background container?